src/EnergyPlus/HVACUnitarySystem.hh

```diff
@@ -489,23 +457,53 @@ namespace HVACUnitarySystem {
 		int FaultyCoilSATIndex;  // Index of the fault object corresponding to the coil
 		Real64 FaultyCoilSATOffset; // Coil SAT sensor offset
 		
+		// variables used in SZVAV model:
```

Comment:
I moved all member variables used in SZVAV to here so that we know which ones are needed in the base class. Each of these variables have the same name in HVACUnitarySystem and PackagedTerminalHeatPump.

Reply:
:+1: !